src/main/java/org/folio/edge/core/Constants.java

```diff
@@ -24,6 +24,13 @@ private Constants() {
   public static final String SYS_REQUEST_TIMEOUT_MS = "request_timeout_ms";
   public static final String SYS_API_KEY_SOURCES = "api_key_sources";
   public static final String SYS_RESPONSE_COMPRESSION = "response_compression";
+  public static final String SYS_SSL_ENABLED = "ssl_enabled";
+  public static final String SYS_KEYSTORE_PATH = "keystore_path";
+  public static final String SYS_KEYSTORE_PASSWORD = "keystore_password";
+  public static final String SYS_KEYSTORE_TYPE = "keystore_type";
+  public static final String SYS_KEYSTORE_PROVIDER = "keystore_provider";
+  public static final String SYS_KEY_ALIAS = "key_alias";
+  public static final String SYS_KEY_ALIAS_PASSWORD = "key_alias_password";
```

Comment:
I am thinking that ENUM would be better approach because of two reason
1. It will be type safe 
2. When using this fields we can just import one emum and use its fields instead of importing all fields as static import

Reply:
resource: https://github.com/AaronLiuIsCool/Effective-Java-Third-Edition-Reading-Notes/blob/master/src/chapter6_Enums_and_Annotations/Enums_and_Annotations.md#item-34-use-enums-instead-of-int-constants

Reply:
this class Constants used in almost all edge-modules and I wanted to implement adding ssl/tls support with minimal changes without adding new structures, expressionally that we would need to backport these changes to Quesnelia.

Reply:
Yes, it make sense, in this case, we can leave it as it was